_data/versions.yml

```diff
@@ -1355,7 +1355,7 @@ master:
   note: ""
   components:
      felix:
-      version: 2.6.0-8-gecbd512d
+      version: 2.6.0-95-g6e89a42f 
```

Comment:
Most recent felix tag, check. I guess we need a felix release in order to bump the container prefix to 3.0.0-alpha1

Reply:
I'm not so worried about cutting releases of things at this point - we have automated and tagged builds of master which is good enough for now while we're still getting things stable.

Reply:
We can just keep this file up-to-date and then we won't have to cut / re-cut so many releases while we're stabilizing. Once we have something that works we can cut a release :)